pyscriptjs/src/utils.ts

```diff
@@ -111,10 +111,14 @@ export function joinPaths(parts: string[], separator = '/') {
 }
 
 export function createDeprecationWarning(msg: string, elementName: string): void {
+    createSingularWarning(msg, elementName);
+}
+
+export function createSingularWarning(msg: string, sentinelText: string): void {
     const banners = document.getElementsByClassName('alert-banner py-warning');
     let bannerCount = 0;
     for (const banner of banners) {
-        if (banner.innerHTML.includes(elementName)) {
+        if (banner.innerHTML.includes(sentinelText)) {
```

Comment:
Should `sentinelText` be called `ElementName` instead? Or `elementText`? 🤔

Reply:
This is the part of what I'm hoping to use to make this function more general. The previous `createDeprecationWarning` is really just looking for the given `elementName` inside the provided msg string... but `elementName` could be any string - an element name, a common Python error we don't want duplicate warnings of... the change in Parameter name reflects that. 

I left `createDeprecationWarning()` as its own function (and now a wrapper around `createSingularWarning`) since there was some discussion of de-duplicating that error based on classes instead of strings, so it's easy to swap in that functionality later if desired.

Reply:
That makes sense thanks 😄